### ISSUE TITLE
UI Placement Options/Dynamic Location

### ISSUE THREAD

**Issue by Terraphice** [tr7zw/ItemSwapper#3]
Obligatory 'this is a feature request!'

If we could have the ability to either move the hotbar switcher UI or select from some more preset locations using whatever configuration library is convenient, I'd appreciate that.

I've always liked the placement to the right of the hotbar that a lot of clock/compass mods use. I'd also really like if it could be directly above whatever hotbar slot you're using at the time, as a bit more of a dynamic option. (With some sort of Raised support, ideally.)

Maybe I'm just missing a config file somewhere. In either case, built-in Mod Menu configuration would be sweet regardless of the method.

**Comment by tr7zw** [comment]
Mod Menu is already in it, maybe you tested using the 0.1.0 version. We are still trying to figure things out, and I guess having different positions should be doable. One issue: This heavily is based on the GUI scale. The "default" scales leave no room to put this anywhere else really.

**Comment by Terraphice** [comment]
Oh yeah, I didn't see anything in the changelog about config support. Nice to know that's a thing now then.

Hm. Perhaps a collapsed menu style then? Like, you press the hotkey and then scroll with the mouse wheel to change the item in that hotbar position, clicking to confirm the switch. Then it wouldn't have any impact on the screen space available.

**Comment by tr7zw** [comment]
But that's already in there? Scrolling works and clicking to close works. Also after a bit of a talk, going to push customization like positioning back to when the design is more finalized.

**Comment by Terraphice** [comment]
I meant a style where the scroll menu doesn't appear above the hotbar at all. It simply replaces that exact single spot in the hotbar with the other items as you scroll. 1 item viewable at a time, replacing the original item in that space but not actually changing it until you click to confirm.

I understand waiting on customization options, so I suppose now what I'm proposing is a potential method for a cleaner selection menu. (But probably not one everyone would like.)

**Comment by tr7zw** [comment]
The scrolling is now a lot more visual + at a different position, and important things like potion types or item names are shown. I don't think this will be part of the current iterations. Closing this issue for now, since it applies to a really old version at this point. Give the new build a try, and potentially open a new issue with ideas based on that build.
